spec/controllers/api/v3/blood_sugars_controller_spec.rb

```diff
@@ -1,36 +1,33 @@
 require "rails_helper"
 
 RSpec.describe Api::V3::BloodSugarsController, type: :controller do
+  before { Flipper.enable(:regions_prep) }
```

Comment:
Need to test for both states of the flag here like above.

Reply:
Addressed in https://github.com/simpledotorg/simple-server/pull/1333/commits/f343427699cf80a265bfc15ae0902876e78cd5d1?diff=unified&w=1.